lib/runctl.js

```diff
@@ -155,6 +172,29 @@ function onRequest(req, callback) {
   }
 }
 
+function requestAllTargets(req, callback) {
```

Comment:
Wait, so these commands actually make *live* changes to the worker process env? That is kind-of handy, I guess, but on the other hand, most env vars are consulted on initialization/startup, and never looked up again, so while this will literally do what it says, it won't actually have any effect to, for example, set `DEBUG` or to set `NODE_ENV=production`... But, it is a primitive operation, so if we are going that way, perhaps the usage should at least make clear that you may need to do a `restart` after changing the env to force it to be picked up?